src/modules/keyboardmanager/ui/EditKeyboardWindow.cpp

```diff
@@ -275,6 +280,10 @@ void createEditKeyboardWindow(HINSTANCE hInst, KeyboardManagerState& keyboardMan
         {
             isSuccess = KeyboardManagerHelper::ErrorType::SaveFailed;
         }
+        if (successfulRemapCount != 0)
+        {
+            Trace::KeyRemapCount(successfulRemapCount);
+        }
```

Comment:
Maybe we should log when there are 0 remaps as well. This would mean that the user cleared all the remappings right?

Reply:
Also, could we log which keys were remapped? That way we would know which are the most popular ones.

Reply:
@saahmedm what are your thoughts on this? I think an issue with logging exact keys that are remapped, might be that it is too much information to be taken from a user? Or this could be a v1 telemetry addition if it's fine to do that.

Reply:
I agree. We can rethink for v1, but for now I think its out of scope

Reply:
we can determine if there are zero by saying for all users, how many don't have one.